app/assets/javascripts/components/timeline/week.jsx

```diff
@@ -76,7 +76,11 @@ const Week = createReactClass({
     const dateCalc = new DateCalculator(this.props.timeline_start, this.props.timeline_end, this.props.index, { zeroIndexed: false });
     let weekDatesContent;
     let meetDates;
-    if (this.props.meetings && this.props.meetings.length > 0) {
+    let meetingDate;
+    if(this.props.meetings){
+      meetingDate=this.props.meetings[0].trim().match(/\(\s*(\d{2}\/\d{2})\s*\)/)[1];
```

Comment:
This is going to be very hard to interpret, with the regex in particular. If a calculation like this is necessary, it should be extracted to well-named method with comments to explain the intended behavior.

Reply:
@ragesoss  I have updated the regex in the well-named component and added proper comments to enhance the intended behaviour